packages/inference/package.json

```diff
@@ -52,9 +52,11 @@
 		"check": "tsc"
 	},
 	"dependencies": {
-		"@huggingface/tasks": "workspace:^"
+		"@huggingface/tasks": "workspace:^",
+		"handlebars": "^4.7.8"
```

Comment:
or we could have used https://www.npmjs.com/package/@huggingface/jinja maybe 🤯

Reply:
cc @xenova

Reply:
That would be awesome! 🤩 Although the library was originally designed for ChatML templates, the set of available features should be large enough for these templates.

Maybe @Wauplin can explain what set of features would be required? 👀

Reply:
basically just `if`s  and variable replacement from what i've seen

Reply:
handlebars has pretty much a feature set of 0.00

Reply:
yea I'm really wary of adding more deps, if we can use jinja it would be great (and we could use jinja for more things)

Reply:
Will update my PR tomorrow in that direction. As Julien said, I'm not using anything fancy at all so jinja will be more than enough for the job

Reply:
Sounds good! Let me know if I can help in any way 🫡

Reply:
I replaced the handlebars dependency by huggingface/jinja and it works like a charm! https://github.com/huggingface/huggingface.js/pull/1255/commits/86e787afd272f3a4a0c8e56a224cfe36a8b6490d Thanks for the package @xenova! :hugs:

Reply:
well I now have `error Couldn't find package "@huggingface/jinja@^0.3.3" required by "@huggingface/inference@*" on the "npm" registry.` in the CI though jinja 0.3.3 is available on https://www.npmjs.com/package/@huggingface/jinja :thinking: